src/cozmo/objects.py

```diff
@@ -365,6 +398,12 @@ def set_lights_off(self):
         self.set_lights(lights.off_light)
 
 
+class Charger(ObservableObject):
+    '''Cozmo's charger object, can be seen and driven towards'''
```

Comment:
Maybe reword: "Cozmo's charger object, which the robot can see and drive toward."